app/frontend/src/components/Card.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import { Typography } from "@material-ui/core";
+import { Meta, Story } from "@storybook/react";
+import React from "react";
+
+import Button from "./Button";
+import {
+  Card,
+  CardActionArea,
+  CardActions,
+  CardContent,
+  CardMedia,
+} from "./Card";
+
+export default {
+  title: "Components/Simple/Card",
```

Comment:
Is this simple or compound

Reply:
I'd say compound based on the number of sub-components that are used with this, since `Card` without any of them is a bit useless on its own?